onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -377,7 +369,6 @@ def _where_input_wrangler(
     "full": core_ops.aten_full,
     "full_like_dtype": core_ops.aten_full_like_dtype,
     "full_like": core_ops.aten_full_like,
-    "gather": (core_ops.aten_gather, _gather_input_wrangler),
```

Comment:
The whole test is deleted

Reply:
lol, that explains why they are passing!